macros/L10nErrors.ejs

```diff
@@ -1,23 +1,30 @@
 <%
- switch(env.locale) {
-    case "ru":
-        translatedStr = "Переведено";
-        errorsStr = "Ошибки Kumascript";
-        inprogressStr = "Частично переведено";
-        woParentStr = "Не связано со страницей en-US";
-    break;
-    case "fr":
-        translatedStr = "Pages traduites";
-        errorsStr = "Erreurs Kumascript";
-        inprogressStr = "Partiellement traduites";
-        woParentStr = "Non liées à une page en-US";
-    break;
-    default:
-        translatedStr = "Translated";
-        errorsStr = "Kumascript errors";
-        inprogressStr = "Partially translated";
-        woParentStr = "Not linked to an en-US page";
- }
+var str = mdn.localString({
```

Comment:
It is regrettable that seems no one to have a testing / review environment before merge commits.